cmake/PACE_Docs.cmake

```diff
@@ -13,7 +13,7 @@ if (sphinx-build)
       )
   else()
     add_custom_target(denullify-docs
-      COMMAND Foreach($f in Get-ChildItem -Path '${DOCS_DIR}/build/html' -Filter *.html) {(Get-Content $f.FullName) | Where-Object {$_ -notmatch '\[NULL\]'} | Set-Content $F.FullName}
+      COMMAND powershell -command "Foreach($f in Get-ChildItem -Path '${DOCS_DIR}/build/html' -Filter *.html) {(Get-Content $f.FullName) | Where-Object {$_ -notmatch '\[NULL\]'} | Set-Content $F.FullName}"
```

Comment:
I'd add `-ExecutionPolicy Bypass` to the PowerShell command https://stackoverflow.com/questions/56726926/running-powershell-script-with-cmake-through-visual-studio-without-changing-syst